packages/apps/kadena-docs/src/components/Layout/components/styles.ts

```diff
@@ -0,0 +1,208 @@
+import { darkTheme, styled, StyledComponent } from '@kadena/react-components';
+
+import Link from 'next/link';
+
+export const Wrapper: StyledComponent<'div'> = styled('div', {
+  maxWidth: '1440px',
```

Comment:
Should we add this to the theme sizes? If it is used in multiple places it could be useful to move it there so it can be changed easily and reused